### PR TITLE
hyperion.ng bump to 9799fae

### PR DESCRIPTION
**Pull request by danielfmo**
Fix hyperion.start
Fix avahi dependency

### CODE REVIEW

**Comment by danielfmo** [comment]
Notes: 
 - New version of hyperion.ng does not use $ADDON_HOME but rather "/storage/.hyperion"
 - hyperiond accepts as argument the filename of the config file under /storage/.hyperion/config, default value "hyperion_main.json";
 - If "hyperion_main.json" does not exist, it is created automatically and populated with default values
 - IMPORTANT: Python3 still needs to be added to target by other means

File: packages/addons/service/hyperion.ng/source/bin/hyperiond.start [hunk 1]
```diff
@@ -21,12 +21,7 @@
 
 oe_setup_addon service.hyperion.ng
 
-if [ ! -f "$ADDON_HOME/hyperion.config.json" ]; then
-  mkdir -p $ADDON_HOME
-  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/hyperion.config.json
-fi
-
-exec hyperiond $ADDON_HOME/hyperion.config.json &
+exec hyperiond &
```
Comment (Ray-future) [inline packages/addons/service/hyperion.ng/source/bin/hyperiond.start]: Can you elaborate why this is necessary? Why not adjust https://github.com/danielfmo/LibreELEC.tv/blob/f664a7a4486b733fb488b82b04b0b19c5b7cefcb/packages/addons/service/hyperion.ng/source/config/hyperion.config.json.sample

**Review by Ray-future — COMMENTED** [review]
Also can you please change the commit message. Change it to something like hyperion.ng: bump to..

File: packages/addons/service/hyperion.ng/source/bin/hyperiond.start (hunk 1 above)
Comment (danielfmo) [reply packages/addons/service/hyperion.ng/source/bin/hyperiond.start]: As I stated in my previous post, hyperiond arguments have changed and the way it manages the configuration file.

I just found how to keep the previous behaviour, I'll try it and update the comit

**Comment by danielfmo** [comment]
Done,

It is working as before, had to re-wire the hyperion.config.json.sample as it was completely outdated.
Now it works flawlessly for new users (config can then be edited with the web interface) and previous config is kept if existing.

Still missing a way to add Python3 to the build

File: packages/addons/service/hyperion.ng/source/bin/hyperiond.start [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
 
-if [ ! -f "$ADDON_HOME/hyperion.config.json" ]; then
+if [ ! -f "$ADDON_HOME/config/hyperion.config.json" ]; then
   mkdir -p $ADDON_HOME
-  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/hyperion.config.json
+  cp $ADDON_DIR/config/hyperion.config.json.sample $ADDON_HOME/config/hyperion.config.json
 fi
 
-exec hyperiond $ADDON_HOME/hyperion.config.json &
+exec hyperiond --rootPath $ADDON_HOME hyperion.config.json &
```
Comment (Ray-future) [inline packages/addons/service/hyperion.ng/source/bin/hyperiond.start:29]: Missing slash? Or is that intended?

File: packages/addons/service/hyperion.ng/source/bin/hyperiond.start [hunk 3]
```diff
@@ -21,14 +21,14 @@
...
 echo $! > /run/hyperiond.pid
 
 if [ -f "/usr/bin/Xorg" ]; then
   exec hyperion-x11 &
-fi
+fi
```
Comment (Ray-future) [inline packages/addons/service/hyperion.ng/source/bin/hyperiond.start:34]: Whitespace after "fi".

**Review by Ray-future — COMMENTED** [review]
Just a few nitpicks. Overall a good job! Thanks.

File: packages/addons/service/hyperion.ng/source/bin/hyperiond.start (hunk 2 above)
Comment (danielfmo) [reply packages/addons/service/hyperion.ng/source/bin/hyperiond.start:29]: It is intended, there are two arguments now. 
root path, plus the configuration file name.
I tested the following use cases:
 - new user to hyperion.ng (no previous configuration/userdata)
 - user that updates from previous versions of hyperion.ng (hyperion.config.json does exist)

All are working

File: packages/addons/service/hyperion.ng/source/bin/hyperiond.start (hunk 3 above)
Comment (danielfmo) [reply packages/addons/service/hyperion.ng/source/bin/hyperiond.start:34]: mistake, will fix

Comment (Ray-future) [reply packages/addons/service/hyperion.ng/source/bin/hyperiond.start:34]: No it was actually my mistake. I thought there was an additional whitespace. Can you remove that again. Sorry for the inconveniences

**Comment by Ray-future** [comment]
Thanks. I have some ideas for python 3.

**Pull request merged**